src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThread.java

```diff
@@ -151,62 +152,59 @@ public StepExecution getStep() {
     @Nonnull Outcome runNextChunk() throws IOException {
         assert program!=null;
 
-        while (true) {
```

Comment:
Reviewers need to look [here](https://github.com/jenkinsci/workflow-cps-plugin/pull/46/files?w=1#diff-1a70e438dd28d7a6307fc2bfb55904e8L154). Basically almost all of this was just reindentation.